web/modules/custom/collection/config/install/rdf_entity.rdfentity.collection.yml

```diff
@@ -1,11 +1,10 @@
 langcode: en
 status: true
 dependencies: {  }
-third_party_settings: {  }
 name: Collection
 rid: collection
 description: 'An Asset Repository stores descriptions of Assets and Asset Distributions.'
 label_singular: collection
 label_plural: collections
 label_count:
-  no_count_capitalize: "Collection\x03Collections"
+  no_count_capitalize: !!binary Q29sbGVjdGlvbgNDb2xsZWN0aW9ucw==
```

Comment:
I understand how this has landed here. However, I prefer the former version which is, at least, human readable. But the whole file change makes no sense. Even the config export is touching a lot of files we should keep in PR only the relevant changes so that the QA is not polluted by noise.